### ISSUE TITLE
`firebase database:set --confirm` reports "STDIN input not available for Windows"

### ISSUE THREAD

**Issue by tohagan** [firebase/firebase-tools#1221]
### [REQUIRED] Environment info

**firebase-tools:**  6.7.1

**Platform:** Windows 10
**Shell**: bash (Mingw)

### [REQUIRED] Test case

     $ echo ./some/path/TEST.json | firebase database:set //some/path/TEST --confirm

NOTE: the path required two slashes // due to another BUG that's already been reported.

### [REQUIRED] Steps to reproduce

1.  echo ./some/path/TEST.json | firebase database:set //some/path/TEST --confirm

2. Outputs

      Error: STDIN input is not available on Windows.

### [REQUIRED] Expected behavior

No Error

### [REQUIRED] Actual behavior

STDIN error 

So basically there is no way using the current CLI to automate importing JSON files.
There are 2 BUGS here actually, one previously reported.

**Comment by tohagan** [comment]
You also cannot run ...
    
     $ firebase database:set //course/en/TEST ./course/en/TEST.json --confirm

... as you'd since the `--confirm` forces you to use STDIN which is broken on Windows.

**Comment by tohagan** [comment]
For my fellow Firebase devs who stumble on this issue ... this tool might be more useful:
- https://github.com/FirebaseExtended/firebase-import/

**Comment by tohagan** [comment]
NodeJs since 8.3.0 can read STDIN on Windows (fix: https://github.com/nodejs/node/issues/10836).

You could remove your [STDIN constraint](https://github.com/firebase/firebase-tools/blob/d8e0072dd46b9417155d5bdbd260a797a9b836fc/src/utils.ts#L116) by testing `if(IS_WINDOWS && process.version < "v8.3.0") and recommend a Node upgrade.

**Comment by tinaliang** [comment]
Thank you for the feedback @tohagan! Can you confirm that upgrading to Node version 8.3.0 fixes the problem for you? If so, we always welcome PRs, and I would be happy to review one from you! Please refer to [contribution.md](https://github.com/firebase/firebase-tools/blob/master/.github/CONTRIBUTING.md) for how to make contributions to the Firebase CLI.

**Comment by mbleigh** [comment]
Note that to set the contents of a file, you don't need to pipe:

    firebase database:set --confirm /path/to/set file_to_set.json

The above will read the contents of file_to_set.json and set them to the database at the specified path.

**Comment by tohagan** [comment]
That now works!  Thanks.
